os-ui/src/component/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+interface ButtonProps{
```

Comment:
better to group this kind of common elements into the separate folder, so we know they can be used in different components

Reply:
This is component is susceptible to change when updating the design of landing page and sign up form. When I update them I will remove all interfaces to the types folder where I have put the other interfaces